docs-md/concepts/queries/push.md

```diff
@@ -8,26 +8,44 @@ keywords: ksqldb, push, query, select
 
 ![Illustration of a push query](../../img/ksqldb-push-query.svg)
 
-Push queries are a form of query issued by a client that subscribe to a result
+A push query is a form of query issued by a client that subscribes to a result
 as it changes in real-time. A good example of a push query is subscribing to a
-particular user’s credit score. The query requests the value of the credit
-score. Because it's a push query, any change to the credit score is "pushed"
-to the client as soon as it occurs over a long-lived connection. This is useful
-for building programmatically controlled microservices, real-time apps, or any
-sort of asynchronous control flow.
+particular user's geographic location. The query requests the map coordinates,
+and because it's a push query, any change to the location is "pushed" over a
+long-lived connection to the client as soon as it occurs. This is useful for
+building programmatically controlled microservices, real-time apps, or any sort
+of asynchronous control flow.
 
 Push queries are expressed using a SQL-like language. They can be used to query
-either streams or tables for a particular key. They’re executed by running over
-the ksqlDB REST API. The result of a push query isn't persisted to a backing
-{{ site.ak }} topic.
+either streams or tables for a particular key. Also, push queries aren't limited
+to key look-ups. They support a full set of SQL, including filters, selects,
+group bys, partition bys, and joins.
 
 Push queries enable you to query a materialized view with a subscription to
 the results. Push queries emit refinements to materialized views, which enable
 reacting to new information in real-time. They’re a good fit for asynchronous
-application flows. For request/response flows, see
-[Pull Query](pull.md).
+application flows. For request/response flows, see [Pull Query](pull.md).

```

Comment:
> Push queries enable you to query a materialized view

As per #4011, push queries are not limited to subscribing to MVs/Tables!  You can subscribe to the output of _any_ query, including on that returns a stream, not a table/mv.